cypress/e2e/attributes.page.cy.ts

```diff
@@ -9,16 +9,23 @@ describe('check for existence of attributes page', () => {
     cy.get('[data-test="launchAttributesPageButton"]').click()
   })
 
-  // Requirement is that for each field the value in the field should be copied to output when you click the field's button
-  // I am using a single variable and should be using multiple variables
-  // Test - deeper cases - off the happy path 
+  // check to see if the 4 attributes columns exist
+
+  it('should display the four columns', () => {
+    // Assert that the four columns exist in the grid
+    cy.get('ion-grid').within(() => {
+      cy.get('ion-row.header-row')
+        .should('contain', 'Adverb')
+        .and('contain', 'Verb')
+        .and('contain', 'Preposition')
+        .and('contain', 'Noun')
+    })
+  }) 
 
-  // check for four input fields 
-  // typing something in one of those fields and then hit a button then read value from read only field and compare that value to what was typed into that field
   
-  // check if the four fields exist on the attributes page
-  // How do I check to see if they exist on the attributes page
-  it('check to see if the four fields exist', () => {
+
+  // check if the four fields exist on the create attributes page
```

Comment:
This test may be incomplete. You should finish it, or remove it. Also, the title and comment may not be accurate, as you are testing if the four fields exist in the test above this one.